yarn.lock

```diff
@@ -13760,114 +13760,114 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@rollup/rollup-android-arm-eabi@npm:4.14.3":
-  version: 4.14.3
-  resolution: "@rollup/rollup-android-arm-eabi@npm:4.14.3"
+"@rollup/rollup-android-arm-eabi@npm:4.18.0":
```

Comment:
Some strange changes in `yarn.lock` without changes in `package.json`.
I think you should to remove it